### PR TITLE
update(getting-started): Update Add Service

### PR DESCRIPTION
**Pull request by levfishbluefish**
### Summary

Overhauls the Getting Started "Add a Service and Route" guide

### Changelog

- Updates from deprecated API to Service and Routes for 0.35
- Adds UI images
- Removes outdate video walkthrough (until a new one is recorded)

### CODE REVIEW

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 1]
```diff
@@ -2,71 +2,108 @@
...
 ---
 
-## 1. Add Your Service Using the Admin API or Kong Manager
+### Introduction
 
-If you'd like to use the Admin API, issue the following cURL request to add
-your first Service (Mockbin) to Kong Enterprise:
+This guide walks through the creation and configuration of a 
+**Service and Route** via **Kong Manager** in **Kong Enterprise**. To do this with 
+via the **command line** read Kong's documentation guide
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
via the **command line** read Kong's guide to 
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 2]
```diff
@@ -2,71 +2,108 @@
...
 
-```bash
-$ curl -i -X POST \
-  --url http://localhost:8001/apis/ \
-  --data 'name=example-api' \
-  --data 'hosts=example.com' \
-  --data 'upstream_url=http://mockbin.org'
-```
+For the purpose of this guide, we’ll create a **Service** pointing to the Mockbin 
+API. Mockbin is an “echo” type public website which returns requests back to
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
API. Mockbin is an “echo” type public website that returns requests back to
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 3]
```diff
@@ -2,71 +2,108 @@
...
+the requester as responses. This is helpful for learning how Kong proxies API
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
the requester as responses. This visibility is helpful for learning how Kong proxies API 
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 4]
```diff
@@ -2,71 +2,108 @@
...
+
+### Step 1 - Create a New Service
+
+1. Chose the desired **Workspace** in **Kong Manager** and navigate to the
+**Services** tab under **API Gateway**.
+
+    ![Service Page](https://d92hw8qwgdmef.cloudfront.net/0.35/getting-started/add-a-service/01-service-page.png)
+
+2. Click the **Create a New Service** button to open the **Create Service** form.
+Fill in a **name** and a **URL** for the service.<br/><br/>For example, set the
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
Fill in a **name** and a **URL** for the **Service**.<br/><br/>For example, set the 
```

Comment (h3xar0n) [reply app/enterprise/0.35-x/getting-started/add-service.md]: Also, why are we using HTML breaks here?

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 5]
```diff
@@ -2,71 +2,108 @@
...
+
+    ![Service Page](https://d92hw8qwgdmef.cloudfront.net/0.35/getting-started/add-a-service/01-service-page.png)
+
+2. Click the **Create a New Service** button to open the **Create Service** form.
+Fill in a **name** and a **URL** for the service.<br/><br/>For example, set the 
+name to `test-service` with the URL `http://mockbin.org`.
+
+    ![Create Service Form](https://d92hw8qwgdmef.cloudfront.net/0.35/getting-started/add-a-service/02-service-name.png)
+<br/><br/>Kong Manager also allows **Services** to be configured using **Protocol**,
+**Host**, **Path**, and **Port** (optional). To configure the Service using
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
**Host**, **Path**, and **Port** (optional). To configure the **Service** using
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 6]
```diff
@@ -2,71 +2,108 @@
...
-  "strip_uri": true,
-  "upstream_connect_timeout": 60000,
-  "upstream_read_timeout": 60000,
-  "upstream_send_timeout": 60000,
-  "upstream_url": "http://mockbin.org"
-}
-```
 
-Kong is now aware of your Service and ready to proxy requests.
+3. Create the Service by clicking the **Create Service** button at the bottom of
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
3. Create the **Service** by clicking the **Create Service** button at the bottom of
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 7]
```diff
@@ -2,71 +2,108 @@
...
+the form.<br/><br/>If successful, you will be automatically redirected to the 
+Service's **Overview** page.<br/><br/>The **Service Overview** page displays
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
**Service's Overview** page.<br/><br/>The **Service Overview** page displays 
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 8]
```diff
@@ -2,71 +2,108 @@
...
-  "upstream_send_timeout": 60000,
-  "upstream_url": "http://mockbin.org"
-}
-```
 
-Kong is now aware of your Service and ready to proxy requests.
+3. Create the Service by clicking the **Create Service** button at the bottom of
+the form.<br/><br/>If successful, you will be automatically redirected to the 
+Service's **Overview** page.<br/><br/>The **Service Overview** page displays 
+information and **Activity** on the Service, as well as any **Routes** or
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
information and **Activity** on the **Service**, as well as any **Routes** or 
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 9]
```diff
@@ -2,71 +2,108 @@
...
-  "upstream_url": "http://mockbin.org"
-}
-```
 
-Kong is now aware of your Service and ready to proxy requests.
+3. Create the Service by clicking the **Create Service** button at the bottom of
+the form.<br/><br/>If successful, you will be automatically redirected to the 
+Service's **Overview** page.<br/><br/>The **Service Overview** page displays 
+information and **Activity** on the Service, as well as any **Routes** or 
+**Plugins** that have been connected to the Service.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
**Plugins** that have been connected to the **Service**.
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 10]
```diff
@@ -2,71 +2,108 @@
...
+
+2. To create a **Route** you must provide the **Service** `name` and `id`, this
+will be automatically filled out if the form was accessed via the 
+**Service Overview** page.
+
+    ![Step 7](https://d92hw8qwgdmef.cloudfront.net/0.35/getting-started/add-a-service/07-route-service-id.png)
+<br/><br/>If this field is not automatically filled, the **Service Id** can be
+found via the **Services Overview** page, accessed by clicking on the **Services**
+tab on the side navigation and then clicking the **clipboard** icon next to the
+desired Services **Id** field.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
desired **Service's Id** field.
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 11]
```diff
@@ -2,71 +2,108 @@
...
+
+3. The **Route** will also need a name, and at least one of the following fields:
+**Host**, **Methods**, or **Paths**<br/>To continue with the Mockbin example,
+name the Route `test-route` with the **Host** `example.com`. Once completed,
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
name the **Route** `test-route` with the **Host** `example.com`. Once completed,
```

File: app/enterprise/0.35-x/getting-started/add-service.md [hunk 12]
```diff
@@ -2,71 +2,108 @@
...
+click the **Create Route** button at the bottom of the form.
+
+    ![Create Route Form](https://d92hw8qwgdmef.cloudfront.net/0.35/getting-started/add-a-service/08-route-form-example.png)
+
+4. If created successfully, the page will be automatically redirected back to
+the **Service Overview** page, and the new Route will now appear under the
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/add-service.md]:
```suggestion
the **Service Overview** page, and the new **Route** will now appear under the
```

**Comment by levfishbluefish** [comment]
> Mostly SG. Could we remove `<br />` and use natural line breaks instead?

Unfortunately, no. This is a bit of a Markdown hack to allow multiple lines and images to be counted under one "bullet"

**Comment by h3xar0n** [comment]
@levfishbluefish Darn, understood. 😞Thanks for letting me know.

**Pull request merged**